src/helpers/types/is_function.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export default function isFunction(arg: AnyValue): string {
```

Comment:
Maybe I'll address it in a future PR. 
Your idea would still not solve my intention to avoid repeating strings all over the application.
For now I removed this feature to avoid merge confilicts.

Reply:
To avoid string repetition we can use global constants that hold the most-likely-to-be-repeated strings.